packages/mobile-components/README.md

```diff
@@ -0,0 +1,79 @@
+## Name: mobile-components
+
+## Description
+
+The mobile-components package offers a set of wrappers and tools for implementing mobile application design.
+
+## Usage
+
+Install `@dbbs/mobile-components` into your application using yarn.
+
+```bash
+yarn add @dbbs/mobile-components
+```
+
+## Examples
+
+Import the tools from mobile-components package and call with appropriate parameters to utilize them in your application.
+
+For example PaperProvider may be used as a wrapper in the App file. As input parameter `theme` should be provided.
+
+```tsx
+import React from 'react'
+import { PaperProvider, appTheme } from '@dbbs/mobile-components'
+
+export const App = () => (
+  <PaperProvider theme={appTheme}>
+    // ...
+  </PaperProvider>
+)
```

Comment:
```suggestion
import React from 'react'
import { PaperProvider, useDefinedTheme } from '@dbbs/mobile-components'
import { AppNavigator } from '@navigators/app-navigator'

export const App = () => {
  const paperTheme = useDefinedTheme()
  return (
    <PaperProvider theme={paperTheme}>
      // ...
    </PaperProvider>
  )
}
```
I think at this point we can show that we prefer to use the theme from the useDefinedTheme hook, which defines the default theme of the application depending on the system theme